.github/workflows/build_cmake.yml

```diff
@@ -70,7 +73,7 @@ jobs:
           COMMAND cmake
             -S .
             -B build
-            -D CMAKE_BUILD_TYPE=$ENV{BUILD_TYPE}
```

Comment:
Was this line intentionally removed?

Reply:
Yup, must have happened while I was flailing about with cmake. Fixed it.